### ISSUE TITLE
SWC University of Ibadan 2019

### ISSUE THREAD

**Issue by embray** [carpentries/african-task-force#3]
This is a separate issue created for tracking discussion of a joint Software Carpentry / [Sage Days](https://wiki.sagemath.org/days102) workshop that is planned to be held at the University of Ibadan in Nigeria July 15 to 19.

This was discussed in #2, but I am creating a separate issue to track discussion specific to that workshop, as opposed to Carpentries community-building in Nigeria in general (@anelda please let me know if this makes sense).

Abbreviations
-------------

|              |                             |
| ------------ | --------------------------- |
| UI           | University of Ibadan        |
| U. Paris-Sud | Université Paris-Sud        |
| JKU          | Johannes Kepler University  |
| NWU          | North-West University       |
| UniPort      | University of Port Harcourt |
| UKZN         | University of KwaZulu-Natal |

Interested people
-------------------

| Local Organizers       | Affiliation  | GitHub         | Mail-pre @      | Mail-post @    | Need travel funding? | Need visa?  |
| ---------------------- | ------------ | -------------- | --------------- | -------------- | -------------------- | ----------- |
| Deborah O. Ajayi       | UI           | ???            | olayide.ajayi   | mail.ui.edu.ng | N/A                  | N/A         |
| Ini Adinya             | UI           | ???            | iniadinya       | gmail.com      | N/A                  | N/A         |

| EU Organizers          | Affiliation  | GitHub         | Mail-pre @      | Mail-post @    | Need travel funding? | Need visa?  |
| ---------------------- | ------------ | -------------- | --------------- | -------------- | -------------------- | ----------- |
| Erik Madison Bray      | U. Paris-Sud | @embray        | erik.bray       | lri.fr         | yes (ODK provided)   | yes         |
| Viviane Pons           | U. Paris-Sud | @VivianePons   | viviane.pons    | lri.fr         | N/A (not attending)  | N/A         |
| Izabela Faguet         | U. Paris-Sud | @IzabelaFaguet | izabela.faguet  | lri.fr         | N/A (not attending)  | N/A         |
| Samuel Lelièvre?       | U. Paris-Sud | @slel          | Samuel.Lelievre | u-psud.fr      | N/A (not attending)  | N/A        |

| Instructors            | Affiliation  | GitHub         | Mail-pre @      | Mail-post @    | Need travel funding? | Need visa?  |
| ---------------------- | ------------ | -------------- | --------------- | -------------- | -------------------- | ----------- |
| Evans Doe Ocansey      | JKU          | @evansdoe       | eocansey        | risc.jku.at    | yes (ODK provided)  | no (need letter) |
| Yaé Ulrich Gaba        | NWU          | @gabayae       | yae             | aims.ac.za     | yes (ODK provided)  | no (need letter) |

| Need Confirmation      | Affiliation  | GitHub         | Mail-pre @      | Mail-post @    | Need travel funding? | Need visa?  |
| ---------------------- | ------------ | -------------- | --------------- | -------------- | -------------------- | ----------- |
| Osamede Henry          | UI           | @cosamede      | cosamede        | yahoo.com      | N/A                  | N/A         |
| Chinedu Obieze         | UniPort      | @CObieze       | chinedu_obieze  | uniport.edu.ng | ???                  | no          |
| Olatunbosun Agbo-Ajala | UKZN         | ???            | ajalabosun      | gmail.com      | ???                  | ???         |
| Dennis Irorere         | Coven Labs   | @denironyx     | denironyx       | gmail.com      | ???                  | ???         |

**Comment by embray** [comment]
CC @chrispunjoku who has also shown interest in local organizing (thank you!)

**Comment by chrispunjoku** [comment]
This is superb.
As I commented on Issue #2 can a usual Carpentries workshop Web page be additionally created for this? The page will bear 15-16 July specifically, since those are the dates for the Carpentries-like workshop. It'll also link to pages of the Carpentries lessons to be taught.  It's impressive to have got three confirmed instructors (Erik, Evans and Yaé).  We need 2 - 4 helpers depending on the number of participants (going by 1 helper to =/<8 participants). It'll also be good to reach Shaikh Reshamas at WIMLDS (info@wimlds.org) with publicity, and thank you Erik for agreeing to this already.

**Comment by sheraaronhurt** [comment]
Hi Everyone, 

I would love to be of assistance and get this on the webpage. I need for you to submit a [Workshop Request Form](https://amy.carpentries.org/forms/workshop/) and include your [workshop website](https://github.com/carpentries/workshop-template). This will allow us to put it on the website and get you the survey monkey results. 

Please let me know if you need anything else! 
Sher!

**Comment by embray** [comment]
@sheraaronhurt Okay, I'll work on that.

In the meantime, if everyone who I @'d in this table help me fill out the parts that have "???" that would be superb; especially the unconfirmed people.  If you don't have permissions to edit the description just comment here and I'll update it.

@chrispunjoku Were you interested in also attending the workshop, or are you just helping for now with the community-building aspect?

**Comment by evansdoe** [comment]
@embray this is my GitHub account and yes I will need an invitation letter just for administrative reasons as it is likely the secretary at our institute might request for it.

**Comment by evansdoe** [comment]
@embray just a slight typo at the section titled Abbreviations. JKU stands for Johannes Kepler University (for English) or Johannes Kepler Universität (for German). Any of them should be fine.

**Comment by chrispunjoku** [comment]
> @chrispunjoku Were you interested in also attending the workshop, or are you just helping for now with the community-building aspect?

I wasn't planning to attend, considering the good number who showed early interest to teach or help in the workshop, but if Chinedu, Olatunbosun and Dennis wouldn't be present, I'll come. So I mark the dates. It'll however be good to find out what's hindering them from contributing to this issue. Would you like to reach them with personal email?

**Comment by embray** [comment]
@evansdoe thank you, I will update that!

**Comment by denironyx** [comment]
Hi @embray and @chrispunjoku, 

I apologize, for not following up on this conversation. I was away for a while. 

It seems a lot of plans have been set in place. However, if you need assistance with anything. Do not hesitate to let me. 

Moreso, if there is room for one more tutor to make this a success. I will be delighted to come on board, as I have double checked my calendar and I happened to have an open schedule for those days.

**Comment by chrispunjoku** [comment]
Thank you @denironyx . Now that you'd confirmed your availability, please have a look at the table in the first post here by @embray . You're expected to provide some important information.

**Comment by denironyx** [comment]
Oh yeah, @chrispunjoku, I won't be requiring a visa to attend this program and I think provision for accommodation will be just.

**Comment by embray** [comment]
This workshop happened the week before last, so I will go ahead and close this for now.

At the end, although we did teach a few Software Carpentry materials, at the end the time we had was too limited to really make this a Software Carpentry workshop per-se, as opposed to the original purposes which was to focus primarily on SageMath and mathematics software.  I had wanted to do both but just couldn't with the time.

Nevertheless, I did tell many people about Software Carpentry and Data Carpentry, and there was clearly a strong desire to have more workshops like that (or like the one we did have) throughout the country, as well as build a stronger network of instructors and subject-matter experts on which some sustainability could be built for keeping workshops regular. 

It was a great experience overall and we all learned a lot, and hopefully many connections were made, even if not strictly in the name of Software Carpentry itself.  I hope in the future there will be more Software Carpentry workshops in Nigeria, as there's clearly a need.  I'll post a link to my report on the workshop here, in case anyone's interested, once it's ready.
